app/components/Newsfeeds.jsx

```diff
@@ -8,6 +8,7 @@ export default class Newsfeed extends React.Component {
     this.handleArticleChange = this.handleArticleChange.bind(this);
     this.state = {
```

Comment:
if the properties of this object can fit on a line there is no need to break them